upsertGitHubTag/deployment/index.js

```diff
@@ -26,16 +26,17 @@ const verifyGitHub = (req, payload) => {
 };
 
 // Makes a POST request to the given path
-function postEndpoint(path, postBody, callback) {
+function postEndpoint(path, postBody, deliveryId, callback) {
   console.log("POST " + path);
-  console.log(qs.stringify(postBody));
+  console.log(postBody);
 
   const options = url.parse(path);
   options.method = "POST";
   options.headers = {
-    "Content-Type": "application/x-www-form-urlencoded",
+    "Content-Type": "application/json",
     Authorization: "Bearer " + process.env.DOCKSTORE_TOKEN,
     "User-Agent": LAMBDA_USER_AGENT,
+    "X-GitHub-Delivery": deliveryId,
```

Comment:
It's unlikely, and there will only be a small window where it could happen, but I think `deliveryId` could be null or undefined, for events created before API Gateway was updated. Is that OK (I don't know)?